core/trino-main/src/main/java/io/trino/server/ui/OAuth2WebUiLogoutResource.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
Commit message and PR title and description should explain what is a change about.

It is fine to add `Fixes #8672` to PR description to use https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword, but commit message should be about the code change.